tests/setup_cfg_fmt_test.py

```diff
@@ -1115,6 +1112,36 @@ def test_version_classifiers_removed_by_default(tmpdir):
     )
 
 
+def test_deprecated_min_py3_version(tmpdir, capsys):
+    setup_cfg = tmpdir.join('setup.cfg')
+    setup_cfg.write(
+        '[metadata]\n'
+        'name = test\n'
+        'version = 1.0\n',
+    )
+
+    assert main((str(setup_cfg), '--min-py3-version=3.7'))
```

Comment:
I added this test and the functionality back -- planning to drop all these changes in a major release and then in the future after that remove the `--min-py3-version` argument -- can't remove it yet because the warning hasn't appeared in a release